src/toil/job.py

```diff
@@ -1722,3 +1721,185 @@ def convertPromises(kwargs):
             elif isinstance(kwargs.get(r), PromisedRequirement):
                 foundPromisedRequirement = True
         return foundPromisedRequirement
+
+
+def docker_call(job,
+                tool,
+                parameters=None,
+                work_dir='.',
```

Comment:
Small nit: if we're moving this code, I feel like it'd be nice for the default for this to be `None` instead of `'.'`, and if the user passes in `None`, then to call `os.getcwd()` to get the working directory, etc.